department-of-veterans-affairs/vets-api
# NOD and SC | Fix attachment size_range to match Lighthouse's API limits

## Summary

*This work is behind a feature toggle (flipper):*
  - NO
*(Summarize the changes that have been made to the platform)*
  - Due to errors with file uploads to lighthouse, we noticed we needed to update the size_range in BYTES to match that of Lighthouse API.
    - update `def size_range` from `1.byte...100.megabytes` to `1.byte...100_000_000.bytes`
- *(If bug, how to reproduce)*
  - add a file size in between 100,000,000 and 104,857,600 bytes and notice a validation error DOES NOT appear
- *(What is the solution, why is this the solution?)*
  - Update `size_range` on backend so that files of that range don't cause file upload upstream errors after a form has been submitted
- *(Which team do you work for, does your team own the maintenance of this component?)*
  - Benefits Team 1 - Decision Reviews - Squad 2


## Related issue(s)

- *Link to ticket created in va.gov-team repo
department-of-veterans-affairs/va.gov-team#74457

## Testing done

- *Describe what the old behavior was prior to the change*
  - Files that are between 100,000,000 and 104,857,600 bytes will not display a validation error during the form submission process but will fail upstream. This error accounts for roughly 8% of our downstream evidence upload errors.
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
  - In both NOD and SC - upload evidence with the specified file size to test this bug.


## Screenshots
|         | Before | After |
| ------- | ------ | ----- |
| Supplemental Claims  | <img width="600" alt="SC_File-size_Before" src="https://github.com/department-of-veterans-affairs/vets-website/assets/37054305/627fa46c-d056-4c13-ae4c-bea6ca38dac6"> | <img width="700" alt="SC_File-size_After" src="https://github.com/department-of-veterans-affairs/vets-api/assets/37054305/ce5eb23d-0307-465b-aab7-b3b8a0f24b9b"> |
| Board Appeal (NOD) | <img width="600" alt="NOD_File-size_Before" src="https://github.com/department-of-veterans-affairs/vets-website/assets/37054305/0b765dff-6da2-4f6a-9942-2393dac307d0"> | <img width="700" alt="NOD_File-size_After" src="https://github.com/department-of-veterans-affairs/vets-api/assets/37054305/17098ed5-ed57-4c79-9b9b-4b592ba4da4f"> |

## What areas of the site does it impact?
Appeal forms:
- Supplemental Claims
- Notice of Disagreement (Board Appeal)

## Acceptance criteria
